lib/closex/mock_client/fixtures/create_lead.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
Is it different from `lead.json`?

Reply:
It is, it's the same as the one in their docs. It doesn't have to be though. Which would you prefer, to look exactly like the CloseIO example responses or to be uniform in our responses for leads/opportunities/etc?